src/client/scripts/cardModal.js

```diff
@@ -82,5 +61,86 @@ Template.cardModal.events({
     "click span i.fa-caret-down": function(eve){
         eve.toElement.className = "fa fa-caret-right";
         $("ul.collapsible li").hide();
+    },
+    "click .edit-card-button": function(eve){
+        eve.preventDefault();
+        $(eve.toElement).hide();
+        Session.set("editCardMode", true);
+    },
+    "submit .addTags": function(e){
+        e.preventDefault();
+        var newTags = e.target.tags.value;
+        var tags = newTags.split(",");
+        var tagSet = new Set();
+
+        tags.forEach(v => tagSet.add(s(v).clean().titleize().value()));
+        var thisRoom = Rooms.findOne({_id:Session.get("roomCode")});
+        var thisTags = thisRoom.tags;
+
+        thisTags.forEach(v => tagSet.add(v));
+        var tagArray = Array.from(tagSet);
+
+        Meteor.call("addTags", tagArray);
+    },
+    "submit #edit-form": function (e) {
+        e.preventDefault();
+        var id = this._id;
+        var thought = 0, cat = 1, tags = 2;
+        var changes = grabEdits(e);
+        $("#" + id).modal("hide");
+        Session.set("editCardMode", false);
+        Meteor.call("updateCard", id, changes[thought], changes[cat], changes[tags]);
     }
 });
+Template.registerHelper("equals", function (a, b) {
+    return a === b;
+});
+
+function isOwner(_id){
+    var card = Cards.findOne({"_id": _id});
+
+    if (Meteor.user()) {
+        if(Meteor.user().profile.name === card.author){
+            return true;
+        }
+    } else if(Session.get("author") === card.author){
+        return true;
+    }
+    return false;
+}
+
+function validComment(eve){
+    var comment = eve.target.parentNode.previousElementSibling.value;
+    var image = null;
+    var commentToAdd = null;
+
+    if(!comment || comment.length <= MinCommentLen)
+        SnackbarMethods.DisplayMessage("Enter a more valuable comment",
+          DEFAULT_SNACKBAR_TIMEOUT);
+    else{
+        var author;
```

Comment:
we are using this enough I think we should make a global utility function getAuthor();  Thoughts on doing this?